packages/extension/src/components/pages/Login/Login.tsx

```diff
@@ -154,9 +177,17 @@ export const Login: FC = () => {
           });
       }
     } else {
-      setPasswordError('Incorrect password');
+      if (currentAttempts >= maxAttempts) {
+        const TIMESTAMP = Date.now() + 1000 * 60 * 15;
+        setDisableLogin(true);
+        saveInChromeLocalStorage('disabledLoginTimer', TIMESTAMP);
+        setPasswordError('Please try again in 15 min');
```

Comment:
```suggestion
        setPasswordError('Too many attempts, please try again later');
```

because otherwise, the value "15 minutes" will be always displayed, even if 10 minutes have already elapsed